thirdparty/runtime_environments.go

```diff
@@ -58,3 +64,38 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// getOSInfo returns a list of operating system changes (name and version) from the corresponding AMI id.
+func (c *RuntimeEnvironmentsClient) getOSInfo(ctx context.Context, amiId string, page string, limit string) ([]OSInfo, error) {
+	apiURL := fmt.Sprintf("%s/rest/api/v1/image", c.BaseURL)
+	params := url.Values{}
+	params.Set("ami", amiId)
+	params.Set("page", page)
+	params.Set("limit", limit)
+	params.Set("type", "OS")
+	reqURL, err := url.Parse(apiURL)
+	if err != nil {
+		return nil, errors.Wrap(err, "parsing API url")
+	}
+	reqURL.RawQuery = params.Encode()
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, reqURL.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+	request.Header.Add("Content-Type", "application/json")
+	request.Header.Add("Api-Key", c.APIKey)
+	resp, err := c.Client.Do(request)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		msg, _ := io.ReadAll(resp.Body)
+		return nil, errors.Errorf("HTTP request returned unexpected status '%s': %s", resp.Status, string(msg))
+	}
+	var decodedOSInfo []OSInfo
+	if err := gimlet.GetJSON(resp.Body, &decodedOSInfo); err != nil {
+		return nil, errors.Wrap(err, "decoding http body")
+	}
+	return decodedOSInfo, nil
```

Comment:
I think it'd be sufficient to just call it `osInfo`!
```suggestion
	var osInfo []OSInfo
	if err := gimlet.GetJSON(resp.Body, &osInfo); err != nil {
		return nil, errors.Wrap(err, "decoding http body")
	}
	return osInfo, nil
```